SU-SOE/soe_basic
# D8CORE-3248: publication set up for SOE

# READY FOR REVIEW

# Summary
- Publication node styles for SOE on

# Review By (Date)
- 2/11

# Urgency
- Med

# Steps to Test

1. Pull in this change.
2. Verify the SOE styling is there. The obvious spot is the More publications cards. These have the blue underline, bigger/heavier font bolding and the arrow for both internal and external links

# Affected Projects or Products
- SOE theming on Publications

# Associated Issues and/or People
- D8CORE-3248

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
